websocket-ballerina/tests/test_dispatching_error.bal

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2021 WSO2 Inc. (//www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// //www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.runtime as runtime;
+import ballerina/test;
+import ballerina/io;
+
+listener Listener l72 = new(21072);
```

Comment:
* Is thr any specific reason to name a listener variable as this (`172`). Would be better if we could be something like `simpleWsListener`.

Reply:
We have hundreds of listeners created for test cases and it is really hard to find unique names for each and every listener. So naming these in this way is easier to add more test cases in the future. Agree that this is not a good naming convention. But since these are test cases I think that is okay. :)